docs/libcudacxx/ptx/instructions/manual/shfl_sync.rst

```diff
@@ -0,0 +1,23 @@
+
+shfl.sync
+^^^^^^^^^
```

Comment:
Q: Why do we need a dedicated file in a `manual` directory? We currently just document PTX instructions inline (with generated parts from the `generated` directory). I see no strong reason to deviate here.